src/Middleware/ThrottleMiddleware.php

```diff
@@ -195,7 +197,31 @@ protected function _touch(): int
             );
         }
 
-        return Cache::increment($this->_identifier, 1, static::$cacheConfig) ?: 0;
+        return Cache::increment($this->_identifier, $this->_getRequestWeight($request), static::$cacheConfig) ?: 0;
```

Comment:
`Cache::increment()` shouldn't be called if `_getRequestWeight()` returns `0`.

Reply:
That will touch the cache key and prolong the TTL, i think we should keep it